library/ssl_cli.c

```diff
@@ -3102,7 +2925,6 @@ static int ssl_parse_server_key_exchange( mbedtls_ssl_context *ssl )
         /*
          * Compute the hash that has been signed
          */
-#if defined(MBEDTLS_SSL_PROTO_TLS1_2)
         if( md_alg != MBEDTLS_MD_NONE )
         {
             ret = mbedtls_ssl_get_key_exchange_md_tls1_2( ssl, hash, &hashlen,
```

Comment:
I'm not sure how this works, as if only MBEDTLS_SSL_PROTO_TLS1_3 is defined, mbedtls_ssl_get_key_exchange_md_tls1_2() does not exist, as it is gated by MBEDTLS_SSL_PROTO_TLS1_2

Reply:
will double check it after #5355 merged .

Reply:
ssl_cli.c is guard with MBEDTLS_SSL_PROTO_TLS1_2